data/json/mapgen/basement/basement_guns.json

```diff
@@ -4,7 +4,7 @@
     "method": "json",
     "om_terrain": [ "basement" ],
     "//": "used by house_31 and house_32",
-    "weight": 250,
```

Comment:
This basement spawns ludicrous numbers of guns. I tried to compromise to keep it impressive, and part of that is making it rarer.